### PR TITLE
[jsk_baxter_startup] change twitter image in baxter.launch

### PR DESCRIPTION
**Pull request by knorth55**
when `spherical_stereo` is enabled, use the camera topic for twitter.

### CODE REVIEW

File: jsk_baxter_robot/jsk_baxter_startup/baxter.launch [hunk 1]
```diff
@@ -14,6 +14,10 @@
   <arg name="launch_spherical_kodak" default="false" />
   <arg name="launch_spherical_stereo" default="false" />
   <arg name="start_openni" default="false"/>
+  <arg name="realsense_camera_ns" default="realsense_torso" />
+  <arg name="spherical_camera_ns" default="kodak_head" />
+  <arg name="spherical_left_camera_ns" default="elp_head_left" />
+  <arg name="spherical_right_camera_ns" default="elp_head_right" />
```
Comment (pazeshun) [inline jsk_baxter_robot/jsk_baxter_startup/baxter.launch:20]: Why do you enable to change camera names?
In my point of view, those names are used in launch file names (e.g., baxter_***realsense_torso***.launch) and argument names (e.g., launch_***realsense_torso***), so should not be changed.

Comment (knorth55) [reply jsk_baxter_robot/jsk_baxter_startup/baxter.launch:20]: this is just for reuse of the value.
i will change it from default to value

Comment (pazeshun) [reply jsk_baxter_robot/jsk_baxter_startup/baxter.launch]: OK, thank you.
I want to ask one more: is `REALSENSE_CAMERA_NS` good name?
Perhaps in future, baxter may have a second realsense camera.
In that case, that name seems confusing.

Comment (knorth55) [reply jsk_baxter_robot/jsk_baxter_startup/baxter.launch]: i think it is ok, and when we have another one we can change the name.
but what is your recommendation?

Comment (pazeshun) [reply jsk_baxter_robot/jsk_baxter_startup/baxter.launch]: Oh, now I understand these names are the same names as the arguments of included roslaunch.
Changing them also seems confusing.
OK, my question is not good.

Comment (knorth55) [reply jsk_baxter_robot/jsk_baxter_startup/baxter.launch]: so, it is better not to use capital letters?

Comment (pazeshun) [reply jsk_baxter_robot/jsk_baxter_startup/baxter.launch]: I think using capital letters is permissible.
You don't have to change anything more, I think.
Sorry for confusing.

**Review by pazeshun — APPROVED** [review]

**Pull request merged**